pybind11/setup_helpers.pyi

```diff
@@ -0,0 +1,47 @@
+from typing import Any, Iterator, Optional, Type, TypeVar, Union
```

Comment:
Might be best to mark this file and setup_helpers.py prominently with a comment, something like:
IMPORTANT: If you change this file, also review \<the other file\> for matching changes.
That will probably preempt many time-consuming trips through CI to catch mismatches.

Reply:
I would assume if you know enough to edit a `pyi` file, you'd know to edit the `py` file too - but the other direction should have a note for sure. :) I'll add both just to cover all bases.

Reply:
That's a good point and it's fine with me without the comment in the .pyi file, but I really like your wording there!